TodoReview.py

```diff
@@ -150,6 +150,8 @@ def run(self, edit, **args):
 		window = self.view.window()
 		paths = args.get('paths', None)
 		settings = Settings(self.view, args.get('settings', False))
+		if not settings.get('current_file', False):
+			paths = [self.view.file_name()]
```

Comment:
`view.file_name()` can be `None` if the file is unsaved. I don't think the current engine can handle this case, so just returning and doing nothing would probably be appropriate.